packages/cli/package.json

```diff
@@ -118,6 +118,7 @@
     "credit-card": "3.0.1",
     "date-fns": "1.29.0",
     "debug": "3.1.0",
+    "decamelize": "^6.0.0",
```

Comment:
This is a small, no fancy syntax, no dependency node module that can run in an Edge environment.